test/dotnet-back-compat.Tests/GivenThatWeWantToBeBackwardsCompatibleWith1xProjects.cs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using FluentAssertions;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+using System.Linq;
+using System.Xml.Linq;
+
+namespace Microsoft.DotNet.Cli.Build.Tests
+{
+    public class GivenThatWeWantToBeBackwardsCompatibleWith1xProjects : TestBase
+    {
+        [Theory]
+        [InlineData("netcoreapp1.0")]
+        [InlineData("netcoreapp1.1")]
+        public void ItRestoresBuildsAndRuns(string target)
+        {
+
+            var testAppName = "TestAppSimple";
+            var testInstance = TestAssets.Get(testAppName)
+                .CreateInstance(testAppName + "_" + target.Replace('.', '_'))
+                .WithSourceFiles();
+
+            //   Replace the 'TargetFramework'
```

Comment:
Remove this comment. The name of the method below is great documentation of its own intent already.